docs/app/node/commands/index.md

```diff
@@ -1,50 +1,82 @@
 ---
-title: Node app project command reference
+title: Node.js app project command reference
 section: node
 ---
 
-## create
+## `deploy`
 
-- `shopify create node` creates a new Node project in a subdirectory of the current directory.
```

Comment:
I was debating whether to include create on this page, but my thinking is, we might want to document the project-type specific options here (compared to the core commands page).

Reply:
I removed it given that `create` is a command in the global context. I think we should document the options on a global command in the same context as the command itself.

At the moment there aren't any context-dependent differences in the options available to `create` so I'd opt to leave it until that changes (if it ever does).